pontoon/base/tests/models/test_translation.py

```diff
@@ -98,6 +98,44 @@ def test_translation_save_latest_update(locale_a, project_a):
     assert project_locale.latest_translation == newer_approved_translation
 
 
+@pytest.mark.django_db
+def test_translation_save_latest_update_for_system_project(locale_a, system_project_a):
```

Comment:
I have noticed in this file, that at least the `test_translation_save_latest_update` attempts to test multiple scenarios at once. It's more like some saga than a single unit test. Would you mind if I split such tests in this file, to cover exactly one scenario?

Reply:
Nice catch! Feel free to do so.

Reply:
Done.